TypeCobol.LanguageServer.Test/ProcessorTests/DataLayoutProcessorCSVTest.cs

```diff
@@ -35,20 +48,68 @@ private static void DoTestProcessor(string sourceFileName, string expectedFileNa
             }
 
             // Compare to expected
-            var expectedPath =  $"{Path.Combine(folder, expectedFileName)}.txt";
-            var expected = File.ReadAllText(expectedPath);
-            TestUtils.CompareLines(testName, result.ToString(), expected, expectedPath);
+            var expected = testData.Expected;
+            TestUtils.CompareLines(testName, result.ToString(), expected, null);
+        }
+
+        private static (string Arguments, string Expected) ParseContent(string testDataFilePath)
```

Comment:
Ok this is inspired by the refactoring processor tests, we can factorize that: the method merely splits the file using the `----` lines as separator so we can have a generic method that return a list of string and two more speciliazed methods. One returning `(string Arguments, string Expected)` for data layout processor and the other returning `(string OriginalSource, string ProcessorType, string CommandArguments, string ModifiedSource)`for refactoring tests.

Reply:
Done